twake/backend/core/src/Twake/Users/Services/User.php

```diff
@@ -256,6 +257,49 @@ public function login($usernameOrMail, $passwordOrToken, $rememberMe = false, $r
 
     }
 
+    public function generateJWT($user, $workspaces = []){
+
+        $key = $this->app->getContainer()->getParameter("env.jwt_secret");
+
+        $expiration = date("U") + 60 * 60; //1 hour
+        $refreshExpiration = date("U") + 60 * 60 * 24 * 31; //1 month
```

Comment:
Can't you push this in configuration file?

Reply:
Of course 👍